luxus/style.css

```diff
@@ -0,0 +1,197 @@
+	border-radius: var(--wp--custom--input--border--radius) !important; /* Requires !important to override local variables with theme variables. */
+	border-width: var(--wp--custom--input--border--width) !important;  /* Requires !important to override local variables with theme variables. */
+	filter: brightness(0.975);
+	font-size: inherit;
+	color: inherit;
+	transition: border-color 0.15s cubic-bezier(.4,0,.2,1);
+}
+
+.jetpack-contact-form .jetpack-field .jetpack-field__input, 
+.jetpack-contact-form .jetpack-field .jetpack-field__textarea {
+	border-color: var(--wp--custom--input--border--color);
+}
+
+.wp-block-jetpack-contact-form input:not(:placeholder-shown),
+.wp-block-jetpack-contact-form textarea:not(:placeholder-shown),
+.wp-block-jetpack-contact-form input:focus,
+.wp-block-jetpack-contact-form textarea:focus {
+	filter: brightness(1);
+	backdrop-filter: saturate(1);
+}
+
+.wp-block-jetpack-contact-form input:focus,
+.wp-block-jetpack-contact-form textarea:focus {
+	border-color: var(--wp--custom--input--focus--border--color);
+}
+
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) select, 
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label)>input, 
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label)>textarea {
+	padding-top: var(--field-padding); /* Remove unnecessary padding adjustment from Jetpack. */
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .animated-label__label {
+	font-size: inherit;
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .animated-label__label {
+	width: auto;
+	padding: 0 4px;
+	transition: transform 0.15s cubic-bezier(.4,0,.2,1), top 0.15s cubic-bezier(.4,0,.2,1), font-size 0.15s cubic-bezier(.4,0,.2,1);
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field.has-placeholder~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field:focus~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field:not(:placeholder-shown)~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap.grunion-field-select-wrap .animated-label__label {
+	background-color: var(--wp--custom--input--color--background) !important;
+	transform: translateY(-11px) translateX(-4px); /* Moves the label out of the field. */
+}
+
+.contact-form label,
+.wp-block-jetpack-contact-form label,
+.jetpack-field-label .rich-text.jetpack-field-label__input {
+	font-weight: 500;
+}
+
+.contact-form__input-error {
+	font-size: var(--wp--preset--font-size--small);
+}
+
+/*
+ * Decorative border for headings with .wave-underline class.
+ */
+:is(h1, h2, h3, h4, h5, h6).wave-underline::after {
+	mask: url(assets/images/wave.svg) no-repeat center / cover;
+	height: 7px;
+	content: " ";
+	width: 200px;
+	background: var(--wp--preset--color--theme-4);
+	display: block;
+	mask-position: center;
+	position: relative;
+	bottom: -9px;
+	left: calc( 100% - ( 50% + 100px ) );
```

Comment:
Not sure that this is important 🤔  but this won't work with left or right aligned headings. The text will be left/right aligned and the wave will be centered.

Maybe this rule should include something like `:is(h1, h2, h3, h4, h5, h6).has-text-align-center.wave-underline::after`

Reply:
I removed this entirely.